modules/core/src/main/java/org/apache/ignite/internal/processors/service/GridServiceProxy.java

```diff
@@ -192,7 +194,9 @@ else if (U.isToStringMethod(mtd))
                             Service svc = svcCtx.service();
 
                             if (svc != null)
-                                return callServiceLocally(svc, mtd, args);
+                                return svcCtx.statisticsEnabled() ?
```

Comment:
Let's rewrite it using if statement, ternary operator good only for one-liners
```
if (svcCtx.statisticsEnabled())
     return  measureCall(ctx.service(), () -> callServiceLocally(svc, mtd, args), name, mtd.getName());

return callServiceLocally(svc, mtd, args);
    
```

Reply:
Fixed